holidays/countries/saint_pierre_and_miquelon.py

```
@@ -0,0 +1,48 @@
+    parent_entity = France
+    subdivisions = ()  # Override France subdivisions.
+    subdivisions_aliases = {}  # Override France subdivisions aliases.
+    # Cession from the UK on May 30th, 1814.
+    start_year = 1815
+
+    def _populate_public_holidays(self) -> None:
+        self.subdiv = "PM"
+        super()._populate_public_holidays()
+
+
+class SaintPierreAndMiquelon(HolidaysPM):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)